rudderanalytics-client/src/main/java/com/rudderstack/sdk/java/RudderAnalytics.java

```diff
@@ -118,6 +125,7 @@ public void shutdown() {
         private long flushIntervalInMillis;
         private List<Callback> callbacks;
         private String configURL;
+        
 
         Builder(String writeKey, String dataPlaneURI) {
```

Comment:
Probably missed in the previous release: Please rename `dataPlaneURI` to `dataPlaneUrl` to keep it sync with other SDKs

Reply:
Updated accordingly